REFERENCE.md

```diff
@@ -7369,6 +7369,16 @@ X-Forwarded-Host and X-Forwarded-Server) get sent to the backend server.
 
 Default value: `undef`
 
+##### `proxy_requests`
+
+Data type: `Boolean`
+
+Sets the [ProxyErrorOverride Directive](https://httpd.apache.org/docs/current/mod/mod_proxy.html#proxyrequests).
```

Comment:
The link text is incorrect. Should be `[ProxyRequests Directive]`

Reply:
Thanks, fixed that!